drone_migration.yaml

```diff
@@ -1,679 +1,413 @@
 [
-   {
-      "kind": "pipeline",
-      "name": "build packages",
-      "services": [ ],
-      "steps": [
-         {
-            "commands": [
-               "echo $DRONE_RUNNER_NAME"
-            ],
-            "image": "alpine",
-            "name": "runner identification"
-         },
-         {
-            "commands": [
-               "git fetch origin --tags",
-               "git status --porcelain --untracked-files=no",
-               "git diff --no-ext-diff --quiet"
-            ],
-            "depends_on": [ ],
-            "image": "golang:1.20.4",
-            "name": "generate tags"
-         },
-         {
-            "commands": [
-               "yarn install --immutable"
-            ],
-            "image": "node:20-bullseye",
-            "name": "install dependencies"
-         },
-         {
-            "commands": [
-               "export NODE_ENV=production",
-               "echo \"export const GIT_COMMIT = '${DRONE_COMMIT}';\" > src/version.ts",
-               "./scripts/replace-package-json-version ${DRONE_TAG}",
-               "yarn build"
-            ],
-            "depends_on": [
-               "install dependencies"
-            ],
-            "image": "node:20-bullseye",
-            "name": "build frontend packages"
-         },
-         {
-            "commands": [
-               "apt update",
-               "apt install zip",
-               "./scripts/package-and-sign ${DRONE_BUILD_NUMBER}"
-            ],
-            "depends_on": [
-               "build frontend packages"
-            ],
-            "environment": {
-               "GRAFANA_ACCESS_POLICY_TOKEN": {
-                  "from_secret": "GRAFANA_ACCESS_POLICY_TOKEN"
-               }
+  {
```

Comment:
Ooops lint auto magic :)

Reply:
The fix is in main, I've updated your branch 👍